tables/customer-schema.js

```diff
@@ -0,0 +1,10 @@
+
+const joi =require('joi');
+
+const customerSchema=joi.object().keys({
+    ID:joi.string().alphanum().min(5).required(),
+    name:joi.string().required(),
+    surname:joi.string(),
+    birth_year:joi.number().integer().min(1900).max(2009).required()
```

Comment:
I'm seeing different standards of variable casing. We normally stick with `camelCase` in our projects. The main thing is to have some consistency